api/meetings/serializers.py

```diff
@@ -177,13 +156,15 @@ def get_download_link(self, obj):
         if getattr(obj, 'file_id', None):
             submission_file = OsfStorageFile.objects.get(_id=obj.file_id)
```

Comment:
After PageCounter work goes in, this line would change to -
submission_file = OsfStorageFile.objects.get(id=obj.file_id)